bb8/src/inner.rs

```diff
@@ -1,18 +1,15 @@
+use crate::api::{Builder, Executor, ManageConnection, PooledConnection, RunError};
+use crate::internals::{Approval, ApprovalIter, Conn, SharedPool, State};
+use futures_channel::oneshot;
+use futures_util::stream::{FuturesUnordered, StreamExt};
```

Comment:
Please undo this unrelated change. I would like to keep the imports in three separate blocks: `std`, external crates and crate-local (in that order).